### PR TITLE
Add a system configuration page

### PR DESCRIPTION
**Pull request by aledeg**
It allows to modify system configuration from the interface. At the moment, only limits are modifiable. The user limit was removed from the user page and added here along with categories and feeds limits.

See #892 and #680

### CODE REVIEW

**Comment by Alkarex** [comment]
I have not tested yet, but looks good :-) Do you want to put anything more in this pull request?

**Comment by aledeg** [comment]
Thanks!
I cannot think of anything else. If you have to test it, maybe you can find something I should add.

**Comment by marienfressinaud** [comment]
I'm not sure I like the idea of moving user limit: it makes more sense to keep it with user management for me…

**Comment by aledeg** [comment]
For me, user management and system configuration is 2 different things. It feels more natural to have the two separated.
I know that the user limit is linked to the users. But I think it safe to say it's a system configuration parameter.

File: app/views/configure/system.phtml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		</div>
+
+		<div class="form-group form-actions">
+			<div class="group-controls">
+				<button type="submit" class="btn btn-important"><?php echo _t('gen.action.submit'); ?></button>
+				<button type="reset" class="btn"><?php echo _t('gen.action.cancel'); ?></button>
+			</div>
+		</div>
+	</form>
+</div>
```
Comment (marienfressinaud) [inline app/views/configure/system.phtml]: Miss a blank line here

File: app/i18n/cz/admin.php [hunk 2]
```diff
@@ -146,6 +146,15 @@
 		'title' => 'Statistika',
 		'top_feed' => 'Top ten kanálů',
 	),
+	'system' => array(
+		'_' => 'System configuration',
```
Comment (marienfressinaud) [inline app/i18n/cz/admin.php]: Could you add a "TODO: translate" each time you don't know the translation?

**Comment by marienfressinaud** [comment]
Ok, it could make sense too. Let me test (today) and I'll tell you ;). Also, I don't forget your email on the mailing

**Comment by aledeg** [comment]
I was thinking it would be a good idea to include on that page the "authentication method" page inside the system configuration page.
It's also a system configuration and having it on the same page than the one I created in this PR would make sense. Later in the process, if we add a lot of system configuration parameters, it would be nice to have a section dedicated for system configuration as we already have the user configuration.

**Comment by marienfressinaud** [comment]
Don't spend too much time on layout organization: we have to do it but I think it should be discussed on a dedicated ticket, with mockups and design proposals otherwise we will have to change again in 6 months ;).

**Comment by aledeg** [comment]
I wasn't going to change more. That was just a thought.

**Comment by aledeg** [comment]
I fixed the minor issues concerning the code formatting.
I added a configuration item for the instance name.

See #858.

**Pull request merged**